recipes/spans/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "spans" %}
+{% set version = "0.3.0" %}
+{% set sha256= "d17d36dc240525e97eae26ff68b63b8a9dc6581e4668581f6e189c24ea50f201" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/74/73/2d0192f38a8caad9ab439210ef943825312902cc9280c9b73a196b5be719/Spans-0.3.0.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - spans 
+    - spans.types
+
+about:
+  home: https://www.github.com/runfalk/spans
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Continuous set support for Python'
+
+  description: |
+    Spans is a pure Python implementation of PostgreSQL’s range types [1]. Range types are conveinent when working with intervals of any kind. Every time you’ve found yourself working with date_start and date_end, an interval may have been what you were actually looking for.
+  doc_url: http://spans.readthedocs.org/en/latest/
+  dev_url: https://www.github.com/runfalk/spans
+
+extra:
+  recipe-maintainers:
+    - runfalk
```

Comment:
@plieningerweb -- any updates to this? Also when you do update this PR, please pull in the latest version of master.